Kajabi/sage-lib
# [DSS-423] Grid column horizontal alignment

## Description

- 1 - [x] add horizontal alignment options for the grid column subcomponent
- 2 - [x] update docs
- 3 - **note** - this has no effect as a flex child

## Screenshots 
![Screen_Shot_2023-08-09_at_1_51_32_PM](https://github.com/Kajabi/sage-lib/assets/1241836/25cd1610-f3bf-4742-91fb-48278875058f)



## Testing in `sage-lib`

- Visit the [Grid Column react view](http://localhost:4100/?path=/docs/sage-grid-gridcol--default-story)
- Verify that horizontal alignment class are present along with the CSS

## Testing in `kajabi-products`

1. (**LOW**) Add prop `horizontalAlignment` to `Grid.Col`. Currently does not exist in the app


## Related

[DSS-423](https://kajabi.atlassian.net/browse/DSS-423)

[DSS-423]: https://kajabi.atlassian.net/browse/DSS-423?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
